src/components/GenreSelect/GenreSelect.tsx

```diff
@@ -19,11 +19,13 @@ const GenreSelect: FC<GenreSelectType> = ({
 
 	const genreButtons = genres.map((genre) => {
 		const className = genre === selectedGenre ? 'selected' : undefined;
+		const testId = `${genre}-genre-button`;
 		return (
 			<button
 				key={genre}
 				className={className}
 				onClick={() => handleSelect(genre)}
```

Comment:
`handleSelect` function is redundant here:
```suggestion
				onClick={() => onSelect(genre)}
```